src/main/java/frc/robot/Robot.java

```diff
@@ -69,6 +70,19 @@
   private final DifferentialDrive m_differentialDrive =
       new DifferentialDrive(m_motorDriveFrontLeft, m_motorDriveFrontRight);
 
+  // Ball Intake
```

Comment:
Is there a reason for why certain variables in this section, including `photoelectricSensorEnter`, `ballDetectedEnterLastLoop`, etc., are not specified as member variables?

Reply:
Nope. Fixed.